typed/declaration/examples.ipldsch

```diff
@@ -0,0 +1,11 @@
+
+type ExampleWithNullable map {String : nullable String}
```

Comment:
One request, feel free to tell me its a terrible idea, but It might be nice if the schema could always be compacted down into a single line.

Reply:
I think that's slightly nuts but if you want to imagine a regexp that replaces newlines with semicolons, I guess sure.

(Kidding.  I mean "parser that regards semicolons and linebreaks as the same".  Don't quote html+zalgo at me.)